### PR TITLE
use postgres downstream image

### PR DESCRIPTION
**Pull request by ldpliu**
## Summary
Use the postgres downstream image 
## Related issue(s)
https://issues.redhat.com/browse/ACM-10597
Fixes #

### CODE REVIEW

File: operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml [hunk 1]
```diff
@@ -915,4 +917,6 @@ spec:
   provider:
     name: Red Hat, Inc
     url: https://github.com/stolostron/multicluster-global-hub
+  - image: quay.io/stolostron/origin-oauth-proxy:4.9
+    name: oauth-proxy
```
Comment (clyang82) [inline operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml:921]: do not need it

Comment (clyang82) [reply operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml:921]: need to update https://github.com/stolostron/multicluster-global-hub/blob/025c1e4777147f35267bb1361e8326a22f5cb838/operator/Makefile#L229 from `10d` to `12d`

Comment (ldpliu) [reply operator/bundle/manifests/multicluster-global-hub-operator.clusterserviceversion.yaml]: > need to update
> 
> https://github.com/stolostron/multicluster-global-hub/blob/025c1e4777147f35267bb1361e8326a22f5cb838/operator/Makefile#L229
> from `10d` to `12d`

Done

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**